fastly/validators.go

```diff
@@ -27,6 +27,13 @@ func validateLoggingPlacement() schema.SchemaValidateFunc {
 	}, false)
 }
 
+func validateLoggingServerSideEncryption() schema.SchemaValidateFunc {
+	return validation.StringInSlice([]string{
+		string(gofastly.S3ServerSideEncryptionAES),
+		string(gofastly.S3ServerSideEncryptionKMS),
+	}, false)
+}
+
```

Comment:
All validators are unit tested. Please add a table test for your new validator to `fastly/validators_test.go`.